i18n.gemspec

```diff
@@ -19,4 +19,6 @@ Gem::Specification.new do |s|
   s.rubyforge_project = '[none]'
   s.required_rubygems_version = '>= 1.3.5'
   s.required_ruby_version = '>= 1.9.3'
+
+  s.add_dependency 'concurrent-ruby', '1.0.2'
```

Comment:
could we, eventually, not have the version locked down so specifically but instead e.g. `~> 1.0`

Reply:
Relaxed in 5480dd33298dd11b24667f47862f8a063a9fad6f.